spec/implementation_matrix.md

```diff
@@ -13,7 +13,7 @@
 | nethermind   |      |      |      |      |      |      |      |      |      |      |       |
 | revm         | :+1: | :+1: | :+1: | :+1: |  🚧  | :+1: | :+1: |  🚧   |  🚧  |      | :+1:  |
 | solidity     | 🚧   | N/A  | 🚧    | 🚧   | N/A  |      |      |      |      |      |       |
-| vyper        |      |      |      |      |      |      |      |      |      |      |       |
+| vyper        | 🚧   | N/A  | 🚧    | 🚧   | N/A  |      |      |      |      |      |       |
```

Comment:
would a word "stale" be a better fit here? it's not actively worked on. We used "stale" for old EIPs before we started updating them

Reply:
Instead of the WIP icon?

Reply:
```suggestion
| vyper        | stale | N/A  | stale | stale | N/A  |      |      |      |      |      |       |
```